src/builtin.c

```diff
@@ -1784,45 +1782,17 @@ static block gen_builtin_list(block builtins) {
   return BLOCK(builtins, gen_function("builtins", gen_noop(), gen_const(list)));
 }
 
-static int builtins_bind_one(jq_state *jq, block* bb, const char* code) {
```

Comment:
We might end up needing such a function again in the future, FYI.  I'm thinking we should have a convention that two consecutive newlines in `src/builtin.jq` should separate logical units, and then we can compile each of those separately, and use them as binders only if a program needs them.